mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -1033,7 +1036,7 @@ class PopulateParamsXDL : public PopulateParamsBase {
     // XXX FIXME: Ignore KReduction XDLOPS path for forward and backward weight
     // convolution now. These M/NPerBlock combinations will result in lowering
     // errors at tuning.
-    if (param.gemmKPack > 1 &&
+    if (param.gemmKPack > 1 && !dataType.isInteger(8) &&
```

Comment:
@jerryyin I can see you made some fixes to `xdlops_gemm_v2` on KReduction path. Could you revise the comment that we only reject them for types other than i8 now?

Once this PR lands, I can check KReduction path for other data types and perhaps we can remove this XXX block altogether.

Reply:
Sure, I'll revise the comment.

BTW, I was originally planning to do it such that:
 1. get this PR merged
 2. get reduction for other data types bring up
 3. Rewrite this lowering to consolidate the logic between kreduction on/off, kpack on/off
 4. Remove all non-reduction case, and keep only reduction (This is such that MI300 can work)

Just want to make sure we don't happen to work on the same thing later.

Reply:
We would still need to keep the non-reduction case for MI-100/200, so perhaps it's time to tap into the unused `arch` attribute.

Reply:
@whchung Good point. Hopefully I can find a way to suppress non-reduction xdlops for MI300, such that reduction xdlops would be the only ones available. To do this, we indeed need the `arch` field.

Coming to think of it, I think `-xdlops` field is just a application of `arch` field as well (It is almost the equivalent of if (`arch` >= `gfx908`).